starfish/plot/spots.py

```diff
@@ -0,0 +1,138 @@
+from typing import Optional, Tuple
+
+import matplotlib.axes
+import matplotlib.pyplot as plt
+import numpy as np
+
+from starfish.types import Features
+from .style import BACKGROUND_COLORMAP, TARGETS_COLORMAP
+from .util import annotate_axis, dpi_correction, remove_axis_frame, set_default_arguments
+
+
+# TODO use the name of the IntensityTable if it exists for a title
+# TODO ambrosejcarr: currently the colormaps for decoded_image and intensities do not align.
+# however, it should be possible to fix that.
+def spots(
```

Comment:
rename to plot_spots, show_spots, plot_spots_on_background? something more expressive than spots would be nice -- but i'll leave it to you to name it.